retry-backoff/README.md

```diff
@@ -0,0 +1,37 @@
+# retry-backoff
+
+Retry operations with incremental backoff.
+
+---
+
+There's a default timeout set to 30 seconds. If you'd like to set a different
+timeout, you'll need to either pass in options to `useRetryWithBackoff`:
+
+```js
+import { main } from "effection";
+import { useRetryWithBackoff } from "@effection-contrib/retry-backoff";
+
+await main(function* () {
+  yield* useRetryWithBackoff(function* () {
+    yield* call(fetch("https://foo.bar/"));
```

Comment:
This won't be compatible with v4, so let's use function syntax: `yield* call(() => fetch(...))`